.ci/unit-tests/chef.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Consider partitioning by action, not provider. If this was called `rspec.sh` both Puppet and Chef could use it.